src/definitions/modules/rating.less

```diff
@@ -69,14 +69,23 @@
   color: @activeColor;
 }
 
+/* Partially Active Icon */
+.ui.rating .icon.partial.active {
+  background: linear-gradient(to right, @activeColor 0% var(--full), @inactiveColor var(--full) 100%);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
```

Comment:
This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.

https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-fill-color
https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-stroke

According to MDN, those webkit features may be a subject to change.
I think web-standard features should be used.

Reply:
Hi, thanks for taking the time to review my PR. I have updated the CSS to remove all non-standard props other than `-webkit-text-stroke`. Using `text-shadow` unfortunately won't seem to work due to the alpha value of the text color of non-active icons, so it shows through 😕

I have created an additional jsfiddle with text-stroke removed in order to show the effect of removing `-webkit-text-stroke`. 
![image](https://user-images.githubusercontent.com/6123140/67987144-9a582180-fc24-11e9-91e0-ea56ab45565e.png)
https://jsfiddle.net/pqkavxrd
![image](https://user-images.githubusercontent.com/6123140/67987206-be1b6780-fc24-11e9-9c45-3965d52a1131.png)
https://jsfiddle.net/pqkavxrd/1/

While I agree it's not ideal to use CSS features which are currently non-standard, I would like to point out that it is not without precedent in other components, i.e.

`font-smooth` in the flag and icon elements
`-webkit-tap-highlight-color` in search module and button element
`-webkit-overflow-scrolling` in the dropdown module